code/modules/virus2/effect/stage_3.dm

```diff
@@ -520,6 +520,17 @@ datum/disease2/effect/lubefoot/deactivate(var/mob/living/mob)
 	mob.reagents.add_reagent(COLORFUL_REAGENT, 5)
 	mob.reagents.add_reagent(PAISMOKE, 5)
 
+/datum/disease2/effect/cleansmoke
+	name = "Cleaning Syndrome"
+	desc = "Causes the infected to synthesize smoke & space cleaner."
+	stage = 3
+	badness = EFFECT_DANGER_HELPFUL
+
+/datum/disease2/effect/cleansmoke/activate(var/mob/living/mob)
+	to_chat(mob, "<span class='notice'>You feel clean!</span>")
+	mob.reagents.add_reagent(CLEANER, 5)
+	mob.reagents.add_reagent(PAISMOKE, 5)
```

Comment:
I hate every single thing that uses PAISMOKE that's not a pAI

Reply:
And I hate spawning the 3 chems that compose smoke and having them react with something else before poofing into smoke.

Reply:
Keep in mind that PAISMOKE also cleans the mob's reagents, not just makes smoke.

If you have making smoke manually, add a SMOKE reagent.

Reply:
Smoke always clears reagents
That's how it works